mobile/src/navigation/course-module/common/constants.ts

```diff
@@ -1,7 +1,14 @@
 import { MaterialTopTabNavigationOptions } from '@react-navigation/material-top-tabs';
 
-import { AppColor, AppFontFamily } from '~/common/enums/enums';
+import {
+  AppColor,
+  AppFontFamily,
+  CourseModuleScreenName,
+} from '~/common/enums/enums';
+import { TabCourseModuleNavigationItem } from '~/common/types/types';
+import { About, Task } from '~/components/course-module/components/components';
 
+const COUNT_TABS_FOR_HIDE = 1;
```

Comment:
We use this constant across navigation module, so let's move it to src/common/constants/navigation.constants.ts and then re-export it from src/common/constants/constants.ts